src/extension/iSamples_hierarchyFacet.js

```diff
@@ -159,15 +159,15 @@ class HierarchyFacet extends React.Component{
                     </span>) : null}
                     {label}
                 </h5>
-            </header>
-            {expanded ?
-             (  <>
                 <div className="switch">
```

Comment:
Moved the toggle to be outside of the hierarchy facet. With the recent `x-tree-view` dependency updates, it seems that putting it parallel in the same block as it makes the toggle unclickable.